solved_ATM_ex/solved_ATM_ex.py

```diff
@@ -0,0 +1,130 @@
+"""Simulating an ATM. """
+
+import sys
+
+ATM_FILE = 1
```

Comment:
Use more meaningful names, 1 is not representing a file.

Reply:
The number 1 refers to the place of the file as a parameter. I didn't understand how can I give a meaningful name instead.

Reply:
"the place of the file as a parameter"
Use these words :)

Reply:
You mean like this: 
THE_PLACE_OF_THE_FILE_AS_A_PARAMETER = 1
or maybe: 
ATM_FILE =1  # the place of the file as a parameter